translations/setup.tex/pl.po

```diff
@@ -125,6 +127,59 @@ msgid ""
+"    Twój Bohater ma teraz Poziom 1.\n"
+"  \\item Ustaw Kafelki Mapy i inne komponenty wybranego Scenariusza jak przedstawiono w Księdze Misji.\n"
+"  \\item Połóż Plansze Miasta wybranych Frakcji przed sobą, a obok niej Kafelki Budynków Miasta.\n"
+"    Sprawdź, które Budynki z wybranego Scenariusza są już wzniesione i połóż odpowiednie Karty Budynków Miasta na Planszy Miasta.\n"
+"    Rozpatrz natychmiastowe efekty z aktualnie wzniesionych Budynków pod koniec przygotowania gry.\n"
+"  \\item Ustaw wartość przychodu jak wskazano w Scenariuszu poprzez umieszczenie Kostek Frakcji na Torach Przychodu na Planszach Miasta.\n"
+"    Umieść Żetony Populacji, Budowy i Księgi Zaklęć w odpowiednich miejscach na Planszach Miasta.\n"
+"  \\item Pogrupuj Żetony Zasobów na stosy, które będą w zasięgu wszystkich graczy.\n"
+"    Weź Zasoby początkowe wskazane przez rozgrywany Scenariusz i umieść je obok swojej Planszy Miasta.\n"
+"    To jest twoja pula Zasobów.\n"
+"  \\item Pogrupuj pozostałe Żetony na osobne stosy.\n"
+"  \\item Posortuj Karty Statystyk na cztery stosy: Ataku, Obrony, Mocy oraz Wiedzy.\n"
+"    Dla każdej Statystki na \\hyperlink{Herocard}{Karcie Bohatera} weź odpowiadającą jej ilość Kart z każdej puli.\n"
+"  \\item Jeśli twój Bohater Główny jest Bohaterem Mocy \\includesvg[height=10px]{\\svgs/might.svg}, dodaj do swojej Talii 1 kartę Zaklęcia \"Magiczna Strzała\", a jeśli grasz Bohaterem Magii \\includesvg[height=10px]{\\svgs/magic.svg}, dodaj 2 takie Zaklęcia.\n"
+"  \\item Do swojej Początkowej Talii dodaj Kartę \\hyperlink{Ability}{Zdolności} oraz \\hyperlink{Specialty}{Specjalności} Poziomu 1.\n"
+"  \\item Potasuj swoją Początkową Talię i umieść ją zakrytą obok swoich Planszetek Bohatera.\n"
+"    Wyżej wymieniona Talia jest już gotową Talią \\textbf{Mocy i Magii}\\index{Talia Mocy i Magii} waszych Bohaterów Głównych. W tej Księdze Zasad będzie nazywana \\textbf{Twoją Talią}.\n"
+"  \\item Pogrupuj Karty Zdolności, Artefaktów i Zaklęć na 3 zakryte Talie (włączając wszystkie niewykorzystane Zaklęcia \"Magiczna Strzała\") i je potasuj.\n"
+"    Z każdej z tych Talii weź po jednej wierzchniej Karcie i umieść je odkryte obok danej Talii, tworząc 3 osobne Stosy Kart Odrzuconych.\n"
+"  \\item Wybierz \\hyperlink{Difficulty}{Poziom Trudności} Scenariusza i weź Premie Początkowe.\n"
+"  \\item Pogrupuj Jednostki Neutralne na 4 Talie według ich poziomu (\\includesvg[height=10px]{\\svgs/bronze.svg}\\includesvg[height=10px]{\\svgs/silver.svg}\\includesvg[height=10px]{\\svgs/golden.svg}\\includesvg[height=10px]{\\svgs/azure.svg}).\n"
+"    Przetasuj te Talie i zostawcie wystarczającą ilość miejsca na ich Stosy Kart Odrzuconych.\n"
+"  \\item Umieść Planszę Walki w zasięgu wszystkich graczy.\n"
+"    Sprawdź, jakie początkowe Jednostki przydziela ci scenariusz, i umieść je na stosie obok swojej Planszy Miasta, osobno od reszty Jednostek Frakcyjnych.\n"
+"  \\item Umieść Tor Rund obok Mapy Gry, a następnie Czarną Kostkę na polu \"1\".\n"
```

Comment:
```suggestion
"  \\item Umieść Tor Rund obok Mapy Gry, a następnie Czarną Kostkę na polu ,,1''.\n"
```
Swoją drogą, czy mapa gry to komponent?

Reply:
ja bym to tak traktował, bo jest zbudowana wyłącznie z komponentów gry, ale to nie wiem jak inni sądzą